### ISSUE TITLE
Responsive figure to container div

### ISSUE THREAD

**Issue by nghenzi** [matplotlib/ipympl#278]
## Describe the issue

Hi, first of all !! Thanks for taking matplotlib to webpages !! 

I am using a matplotlib figure inside a resizable div in last Ipympl and I would like the matplotlib figure fills all the available space automatically without the handle.  In particular, I need the imshow functionality, due I could not replicate its behaviour with any other plotting library. I read issue #117, #207 and #239, but I could not find a workaround. If I try with version 0.3 the figure fill the width, but not the height. Is there any workaround to use this functionality, maybe other ipympl version? Or somebody can give  me some hint to implement it ? I would be very happy to implement a PR. 

Thanks ! 

## Versions

```
Paste the command output here.
```

**Comment by martinRenou** [comment]
Hi!

This was actually a feature implemented in ipympl 0.3.1, but it was reverted due to the issue you linked to #117, see my comment here: https://github.com/matplotlib/ipympl/issues/117#issuecomment-546221100

You can use ipympl 0.3.1, we might want to have different display modes in ipympl, one being the classical mode, another one being the responsive mode.

> If I try with version 0.3 the figure fill the width, but not the height

You should get away with it by doing:
```
fig.canvas.layout.height = 'auto'
```

**Comment by nghenzi** [comment]
Great !!! 

thank you so much for your fast response !
